### ISSUE TITLE
Focus outline should cover search/button

### ISSUE THREAD

**Issue by jbergstroem** [picturepan2/spectre#199]
<img width="282" alt="screen shot 2017-04-17 at 3 47 39 pm" src="https://cloud.githubusercontent.com/assets/176984/25100114/5333d602-2385-11e7-978d-7de379e7666a.png">

In at least Safari/Chrome, the outline for a `<input type="text">` doesn't cover search which I at least "feel" would be the right thing to do. I obviously know why this is occurring so I guess the issue could be whether (or not) this should be a thing :)

**Comment by jbergstroem** [comment]
I guess this applies to most [input groups](https://picturepan2.github.io/spectre/elements.html#forms) (scroll a bit further).

**Comment by picturepan2** [comment]
@jbergstroem Yes, it is a known issue which doesn't look perfect. There is a way to make the whole component covered by the outline. But more complex code structure.
I will try to make it in the Experimentals first to test whether it is workable.
